### DISCUSSION TITLE
Is there an option available to have a CDN link instead of SVG?

### DISCUSSION THREAD

**Discussion by SharmaTushar1** [simple-icons/simple-icons#9249] [General]
Hi, just came across simple icons a few days ago. I was wondering if there was an option to have a link CDN (as in icons8.com see the image below) instead of an actual SVG saved in your codebase. The reason why one might prefer a link over a file will be:
- Easy to set up. Just copy the link and paste it into your code.
- Cases where the company icon is updated. This is the major one. Say, Twitter recently announced they'll be changing their icon to X. I know in the case of Twitter this change can go either way as of right now but this made me wonder if we can just have a link and can just update the SVG on our cdn link so users won't have to worry about updating their SVG on their codebase.

This might not be the best question to ask as from what I know maintaining a CDN requires constant money to be spent (can be wrong though haven't worked with CDNs of major scale) but I was curious why we're not doing it yet (if this is not the case).

The image I was referring to: 
![2023-08-09 19_10_38-Twitterx Icons, Logos, Symbols – Free Download PNG, SVG](https://github.com/simple-icons/simple-icons/assets/62995185/ec4f163f-3515-4959-9211-fbba5c69d289)

**Comment by adamrusted** [comment]
We do have https://cdn.simpleicons.org/ maintained by @LitoMore, and you can find instructions on its use in our README.md.
Not something we've included on the website as of yet, as you're absolutely correct - hosting the CDN and utilising it across our site would be two very different workloads.

Happy for us to have a discussion around costings of running it 'properly' though, we seem to be in a good position from a cursory glance at OpenCollective.

**Comment by LitoMore** [comment]
If you kindly read through our documentation then you will find: https://github.com/simple-icons/simple-icons#cdn-usage
